services/web/server/src/simcore_service_webserver/exporter/config.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseSettings, Field, PositiveInt
+
+
+class ExporterSettings(BaseSettings):
+    max_upload_file_size: PositiveInt = Field(
+        10, description="size in GB of the maximum projects/import request"
+    )
+    downloader_max_timeout_seconds: PositiveInt = Field(
+        60 * 60,
+        description="maximum timeout for each individual file used in the file_downlaoder",
+    )
+
+    class Config:
+        case_sensitive = False
+        env_prefix = "WEBSERVER_EXPORTER_"
+
+
+exporter_settings = ExporterSettings()
```

Comment:
this should be in the setup, as I mentioned above.  We do not want to have globals. Instead we prefer singletons INSIDE the app. Nevermind, it is minor. I will change it later